### DISCUSSION TITLE
Feedback from first session

### DISCUSSION THREAD

**Discussion by Stoob** [fariapp/fari-app#189] [Feedback]
Having played for 2-3 hours in the Fari app our first game of Fate it worked pretty well.

1. I was able to distribute their characters via a .json file which they all figured out how to upload prior to joining the game
2. The rolling was very intuitive as was the Fate-point system
3. Players enjoyed seeing their characters and aspects.  they wished they could also see their stunts without clicking their character sheet
4.  during the game I found the sort feature worked strangely.  I had a few aspects, and NPC and 3 bad guys.  I am unable to drag and drop to reorder them.  The sort button ended up placing the bad guys in three separate columns. I wish I had taken a screenshot but forgot
5. I noted that the Day/Night icon works independently for each character.  Meaning as GM if I click Night, they PC screens remain day.  I think this may be a bug.  It seems like the GM should set day/night for all characters
6. at the end of the game the biggest complaint of the players is they could not figure out how to export their characters. I instructed them to click 'Fari' in the header, and then click Characters, and then download the .json file.  But this was not clear to them and some struggled

**Comment by RPDeshaies** [comment]
Thanks for your feedback! I'll try to address some of your comments here:

1. Nice! Good to know the process was easy and strait forward
2. Awesome. It's such a crucial part of gameplay and I'm pleased to know it worked well
3. In v4 #187 you will be able to select any section of the character sheet and mark it as a field you you want to see in the cards. Stay tuned
4. Sorry about this but yeah it's not possible to "drag and drop" aspects. Either they are "sorted in the order they were added" or they are "sorted by pinned + categories". It's not ideal but I hope that I will be able to add proper drag and drop functionalities in v4 as well as I'm currently working on adding drag and drop functionalities in the character sheet and so far it's working pretty well
5. The day/night icon is more of a "toggle dark theme or light theme" so I never intended it to work as a "change theme for everyone" though it would be a pretty interesting feature!
6. I would be curious to know more about this. I know I specifically disable the header menu when someone is "in game" but apart from that the flow should be pretty strait forward so if you have more information, please don't hesitate
